dotcom-rendering/src/web/components/PinnedPostLiveness.importable.tsx

```diff
@@ -38,5 +63,45 @@ export const PinnedPostLiveness = ({}) => {
 			);
 		};
 	}, []);
+
+	// calculate duration when user is viewing pinned post
+	// and emit ophan events when the pinned post goes out of view
+	useEffect(() => {
+		if (!pinnedPost) return () => {};
```

Comment:
You don’t need to return an empty cleanup function.

```suggestion
		if (!pinnedPost) return;
```